### ISSUE TITLE
[Bug]: REAL values in scientific notation with positive exponent not serialized correctly

### ISSUE THREAD

**Issue by fkleedorfer** [ThatOpen/engine_web-ifc#695]
### What happened?

For very small or very big values, we get lines like these when we write the model:

`#582=IFCPROPERTYSINGLEVALUE('W\X2\00F6\X0\lbwiderstand_cm6',$,IFCWARPINGCONSTANTMEASURE(1.3E+14.),#474);`
(note the trailing '.' in `1.3E+14.`)

`#556=IFCPROPERTYSINGLEVALUE('Frequenz_GHz',$,IFCINTEGERCOUNTRATEMEASURE(1.3E-07),#428);`
(note no trailing '.' in `1.3E-07.`)

By checking with other IFC tools, I get the impression that using scientific notation is ok, but the exponent must be an INTEGER. After removing any trailing '.' characters, other tools (BIMVision) can read the file.

Summing up: I think that there is a bug with positive exponents only, and the bug is that the exponent is seralized as REAL but should be INTEGER

### Version

0.0.51

**Comment by beachtom** [comment]
Should now be fixed
